### PR TITLE
Remove deprecated apis not available in OpenFin v22

### PR DESCRIPTION
**Pull request by ilzep**
OpenFin v22 has deprecated a number of APIs that are used in this project. Removing references to grouping and notification APIs that are no longer available.

### CODE REVIEW

File: packages/desktopjs-openfin/src/openfin.ts [hunk 1]
```diff
@@ -153,37 +153,19 @@ export class OpenFinContainerWindow extends ContainerWindow {
     }
 
     public get allowGrouping() {
```
Comment (bingenito) [inline packages/desktopjs-openfin/src/openfin.ts:155]: I believe for allowGrouping, getGroup, joinGroup and leaveGroup you can just delete completely as the base ContainerWindow returns false and is not supported.

File: packages/desktopjs-openfin/src/openfin.ts [hunk 2]
```diff
@@ -573,7 +539,7 @@ export class OpenFinContainer extends WebContainerBase {
     }
 
     public showNotification(title: string, options?: NotificationOptions) {
```
Comment (bingenito) [inline packages/desktopjs-openfin/src/openfin.ts:541]: same as grouping methods, the base class already throws not implemented so you can just delete the whole member

File: packages/desktopjs-openfin/src/openfin.ts (hunk 1 above)
Comment (ilzep) [reply packages/desktopjs-openfin/src/openfin.ts]: Good point, removed the methods from this file

**Review by bingenito — APPROVED** [review]
lgtm.  tested PR locally and checks out

**Pull request merged**